fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_crypt.dm

```diff
@@ -602,6 +613,121 @@
 			return
 
 
+/obj/structure/bloodsucker/vassalrack/proc/shapeshift_victim(mob/living/user, mob/living/target)
+	var/datum/antagonist/bloodsucker/bloodsuckerdatum = user.mind.has_antag_datum(/datum/antagonist/bloodsucker)
+	/// To deal with Blood
+	var/mob/living/carbon/human/C = user
+	var/mob/living/carbon/human/H = target
```

Comment:
bad var names use better ones

Reply:
actually why do you set these? you can just have the 2 args be carbon humans from the start